### ISSUE TITLE
Booting Model traits

### ISSUE THREAD

**Issue by juampi92** [envant/eloquent-lockable#4]
Hey man. I just noticed that you are overwriting the boot method on a model trait, which might have conflicts with other packages.

Here's the line: https://github.com/envant/eloquent-lockable/blob/9c0a3d819599e7b236906101169b077767f91d6d/src/Lockable.php#L50

To fix this, Laravel has a `bootTraits` functionality. You can check it out here: https://github.com/laravel/framework/blob/5.8/src/Illuminate/Database/Eloquent/Model.php#L207

Basically replace the method definition for 
```php
  public static function bootLockable()
```

And no need to call any parent.

Cheers!

**Comment by Davidnadejdin** [comment]
> Hey man. I just noticed that you are overwriting the boot method on a model trait, which might have conflicts with other packages.
> 
> Here's the line:
> 
> https://github.com/envant/eloquent-lockable/blob/9c0a3d819599e7b236906101169b077767f91d6d/src/Lockable.php#L50
> 
> To fix this, Laravel has a `bootTraits` functionality. You can check it out here: https://github.com/laravel/framework/blob/5.8/src/Illuminate/Database/Eloquent/Model.php#L207
> 
> Basically replace the method definition for
> 
> ```
>   public static function bootLockable()
> ```
> 
> And no need to call any parent.
> 
> Cheers!

thank you very much
